src/nl/hannahsten/texifyidea/completion/BibtexCompletionContributor.kt

```diff
@@ -63,4 +62,17 @@ open class BibtexCompletionContributor : CompletionContributor() {
             .withLanguage(BibtexLanguage),
         BibtexStringProvider
     )
+
+    private fun registerFileCompletion() = extend(
+        CompletionType.BASIC,
+        PlatformPatterns.psiElement()
+            .inside(BibtexContent::class.java)
+            .withPattern("File completion pattern") { psiElement, _ ->
+                val key = psiElement.firstParentOfType(BibtexTag::class)?.firstChildOfType(BibtexKey::class) ?: return@withPattern false
+                // Currently, the bibsource field is used for file sources, but this is apparently not 'official'
+                key.text == "bibsource"
```

Comment:
what about: `key.text in bibtexFileKeys`?
https://github.com/Hannah-Sten/TeXiFy-IDEA/blob/3464f1bbcb7fcf8bed6e103aaf216addafa6aa5e/src/nl/hannahsten/texifyidea/util/magic/FileMagic.kt#L48

Reply:
Oops, thanks!